Sources/Plot/API/HTMLElements.swift

```diff
@@ -471,3 +471,11 @@ public extension Node where Context: HTMLScriptableContext {
         .element(named: "script", nodes: nodes)
     }
 }
+
+public extension Node where Context == HTML.ScriptContext {
```

Comment:
These are new APIs are for attributes, so they should be defined within `HTMLAttributes.swift`.